packages/cf-component-modal/package.json

```diff
@@ -11,9 +11,9 @@
     "registry": "http://registry.npmjs.org/"
   },
   "dependencies": {
-    "cf-component-heading": "^4.3.1",
     "cf-component-icon": "^2.2.1",
     "cf-style-container": "^1.1.1",
+    "react-addons-css-transition-group": "^0.14.2 || ^15.0.0-0",    
```

Comment:
use https://github.com/reactjs/react-transition-group instead 

https://facebook.github.io/react/blog/#discontinuing-support-for-react-addons